apps/server/src/modules/files/entity/file.entity.spec.ts

```diff
@@ -94,6 +94,41 @@ describe(FileEntity.name, () => {
 		});
 	});
 
+	describe('removCreatorId', () => {
+		describe('when called on a file that contains matching creatorId', () => {
+			it('should properly remove this creatorId', () => {
+				const file = fileEntityFactory.build({
+					ownerId: mainUserId,
+					creatorId: mainUserId,
+				});
+
+				const expectedFile = copyFile(file);
+				expectedFile._creatorId = undefined;
```

Comment:
Please move it to the `setup()` function.

Reply:
done